main_test.go

```diff
@@ -361,6 +361,36 @@ func Test_newConfig(t *testing.T) {
 		assert.Equal(t, 100*time.Second, config.ReconnectionTimeout)
 	})
 
+	t.Run("should return an error when failing to parse RYUK_VERBOSE environment variable", func(t *testing.T) {
+		t.Setenv(verboseEnv, "bad_value")
+
+		config, err := newConfig([]string{})
+		require.NotNil(t, err)
+		require.Nil(t, config)
+	})
+
+	t.Run("should set connectionTimeout with RYUK_VERBOSE environment variable", func(t *testing.T) {
+		t.Setenv(verboseEnv, "true")
+
+		config, err := newConfig([]string{})
+		require.Nil(t, err)
+		assert.True(t, config.Verbose)
+
+		t.Setenv(verboseEnv, "false")
+
+		config, err = newConfig([]string{})
+		require.Nil(t, err)
+		assert.False(t, config.Verbose)
+	})
+
+	t.Run("should set port from env with port flag and RYUK_VERBOSE environment variable", func(t *testing.T) {
+		t.Setenv(verboseEnv, "false")
+
+		config, err := newConfig([]string{"-v"})
```

Comment:
```suggestion
		config, err := newConfig([]string{"-p", "3000"})
```

Should this be the port flag according to the test description:

> should set port from env with port flag and RYUK_VERBOSE environment variable

Reply:
I did not push a commit removing the test, as it's outdated 🤦 Pushed.